src/components/Tables/Table.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import React, { useState, useMemo, useEffect } from "react";
+import DataTable, { TableColumn } from "react-data-table-component";
+import TableFilterBar from "./TableFilterBar";
+
+interface Props {
+    data: { [key: string]: string | number | null }[];
+    headers: { [key: string]: string | null };
+}
+
+const Table: React.FC<Props> = (props) => {
+    const [filterText, setFilterText] = useState<{ [key: string]: string }>({});
+
+    // temporary solution to fix hydration error
```

Comment:
Can you remove 'temporary' - just `Fixes hydration error by re-rendering when the DOM is ready` would be better.
Potentially even leaving the link to the NextJS docs you got this from, or why it is necessary

Reply:
Done!